src/dotnet-toolbox/UninstallCommand.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using DotNetToolbox.Helpers;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace DotNetToolbox
+{
+    public class UninstallCommand : CommandLineApplication
+    {
+        private ToolboxConfiguration _toolboxConfig;
+        public UninstallCommand(CommandLineApplication parent)
+        {
+            Parent = parent;
+            Name = "uninstall";
+            PackageArgument = new PackageArgument(this);
+            Arguments.Add(PackageArgument);
```

Comment:
We should probably support both. I want to avoid dependencies on the structure of the nuget packages folder as much as possible, so I think saving the mapping between the tool name and the package ID in the install file would be a better approach.